lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -437,29 +814,80 @@ void HdVP2Mesh::Sync(
 #endif
     }
 
```

Comment:
We will also need to apply the material consolidation update workaround to geom subset scenario. Here is the reference code:

https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/blob/4d66c9264641c0f68a964f128eb14819a7d88558/pxr/imaging/plugin/hdRpr/mesh.cpp#L288

Reply:
Okay so I need to call SubscribeForMaterialUpdates & UnsubscribeForMaterialUpdates in reasonable places? I will figure out where to add it.

Reply:
This has been fixed by new commits.